css/subject.styl

```diff
@@ -2,9 +2,10 @@
   min-width: 0
 
 .subject
+  margin: 0 auto
+  max-width: 800px
   .subject-container
```

Comment:
Is there anywhere on the site where the stack goes `.subject > .subject-container`? I think these are orphaned styles.